haystack/nodes/answer_generator/openai.py

```diff
@@ -53,6 +54,7 @@ def __init__(
 
         :param azure_deployment_name: The name of the Azure OpenAI API deployment. If not supplied, Azure OpenAI API
                                      will not be used.
+        :param api_base: The base URL for the OpenAI API, defaults to `"https://api.openai.com/v1"`.
```

Comment:
Let's move this to the end of doc strings given that we'll move `api_base` to the end of the arguments.